.github/workflows/publish.yml

```
@@ -0,0 +1,30 @@
+name: Publish
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: '20.x'
```

May as well match our production setup

```suggestion
        node-version: '20.9'
```